Rakefile

```diff
@@ -10,8 +10,51 @@ namespace :design_system do
   end
 end
 
-desc "Rubocop Linting"
-task :rubocop do
-  puts "Running rubocop linter"
-  system("bundle exec rubocop") || raise
+desc "Run all checks in the local context"
+task check: ["ruby:lint", "npm:test"]
+
+namespace :ruby do
+  desc "Lint ruby files"
+  task lint: [:rubocop, :haml_lint]
```

Comment:
Unsure if this'll crop up but you may wanna go `%i[rubocop haml_lint]` here. Might fall foul

Reply:
Fall foul of the style rules? It passes happily on the branch at the moment.